core/src/main/resources/openapi.yaml

```diff
@@ -4796,10 +4796,10 @@
           "currentPage" : {
             "type" : "string"
           },
-          "deletable" : {
+          "editable" : {
             "type" : "boolean"
           },
-          "editable" : {
+          "deletable" : {
```

Comment:
can you please give a bit more details why do we need changes like that within this PR?

Reply:
Yeah this is a bit confusing, there is no actual change here except in ordering.
It is automatically built and unfortunately seems to switch the ordering of a few variables every time, I should stop adding these changes and do a git diff before adding it next time to see if there is anything substantial in it.

Reply:
i wonder if we can be sure whether we could guarantee that the ordering is kept same in case no code changes happen

Reply:
I have an open issue at the moment to develop this further, I can definitely add that in.
We would like to have the ability to get it to add comments and examples aswell as part of the autogeneration

Reply:
ok then please put a link to the issue here

Reply:
Annotate OpenAPI Spec
https://github.com/aiven/klaw/issues/665

Reply:
Annotate OpenAPI Spec
https://github.com/aiven/klaw/issues/665